src/bigquery.ts

```diff
@@ -1278,6 +1280,11 @@ export class BigQuery extends common.Service {
       },
     };
 
+    if (query.jobTimeoutMs) {
```

Comment:
Sorry I missed this one. If this can be 0, we have to check that it's defined as a number.

Reply:
Thanks for catching that!